### DISCUSSION TITLE
GitHub Co-Pilot Support

### DISCUSSION THREAD

**Discussion by slowsmokeyjoe** [community/community#60713] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Does GitHub Support also apply to GitHub Co-Pilot support as mentioned here: https://docs.github.com/en/support/learning-about-github-support/about-github-support#about-github-support? Thanks.

**Comment by AliAlmasi** [comment]
This is what [that page](https://docs.github.com/en/support/learning-about-github-support/about-github-support#about-github-support) said:

> If your account uses a paid GitHub product or you are a member of an organization that uses a paid product, you can directly contact GitHub Support. If your account uses GitHub Free, you can speak to GitHub users and staff on the [GitHub Community discussions](https://github.com/orgs/community/discussions) for most issues, and you can contact GitHub Support to report account, security, and abuse issues.

So... I think they can support it since you're using _a paid GitHub product_, huh?

**Comment by davecheney** [comment, marked as answer]
Yes, you may contact support.

**Comment by RameshGurram1** [comment]
Need support on the GitHub Co-pilot, costing as well.
